README.md

```diff
@@ -0,0 +1,71 @@
+Flask ICU
+=========
+
+[![Build Status](https://travis-ci.org/beavyHQ/flask-icu.svg?branch=retrofit-for-pyicu)](https://travis-ci.org/beavyHQ/flask-icu)
+
+Implements i18n and l10n support for Flask using the industry standard
+ICU.
+
+#### ICU Syntax
+
+Simple example:
+```
+I have {numCats, number} cats.
```

Comment:
as you already use a plural version, maybe pretty it up a bit? with "0 => no cats",  "1 => one cat", "_ => number cats"?